frontend/css/layout.css

```diff
@@ -14,21 +16,18 @@ a {
 
 .account {
     justify-content: flex-end;
-    padding-right: 20px;
     border-bottom: 0.5px solid lightgray;
     font-size: 14px;
```

Comment:
Can we have a variable for the `14px` font size (and converted to rems) in `typography.css`?